api/db/schema.prisma

```diff
@@ -142,13 +146,11 @@ model UploadValidation {
   inputTemplateId     Int
   inputTemplate       InputTemplate @relation(fields: [inputTemplateId], references: [id])
   validationResults   Json?         @db.JsonB
-  validatedAt         DateTime?     @db.Timestamptz(6)
-  validatedById       Int?
-  validatedBy         User?         @relation("ValidatedUploads", fields: [validatedById], references: [id])
   invalidationResults Json?         @db.JsonB
-  invalidatedAt       DateTime?     @db.Timestamptz(6)
-  invalidatedById     Int?
-  invalidatedBy       User?         @relation("InvalidatedUploads", fields: [invalidatedById], references: [id])
+  reviewedAt          DateTime      @default(now()) @db.Timestamptz(6)
+  reviewedById        Int?
+  reviewedBy          User?         @relation(fields: [reviewedById], references: [id])
+  reviewType          ReviewType?
   createdAt           DateTime      @default(now()) @db.Timestamptz(6)
   updatedAt           DateTime      @default(now()) @db.Timestamptz(6)
```

Comment:
before upload
```
[
    UV(review_results: {invalid results here}, review_type: invalid, reviewed_at: yesterday, created_at: yesterday, updated_at: yesterday),
    UV(review_results: {}, review_type: valid, reviewed_at: day_before, created_at: day_before, updated_at: day_before),
]

```

Time of upload
```
[
    UV(review_results: {}, review_type: null, reviewed_at: today, created_at: today, updated_at: today),
    UV(review_results: {}, review_type: invalid, reviewed_at: yesterday, created_at: yesterday, updated_at: yesterday),
    UV(review_results: {}, review_type: valid, reviewed_at: day_before, created_at: day_before, updated_at: day_before),
]

```

Time of upload + 5 minutes
```
[
    UV(review_results: {result1, result2}, review_type: valid, reviewed_at: today, created_at: today, updated_at: today),
    UV(review_results: {}, review_type: invalid, reviewed_at: yesterday, created_at: yesterday, updated_at: yesterday),
    UV(review_results: {}, review_type: valid, reviewed_at: day_before, created_at: day_before, updated_at: day_before),
]

```

Reply:
Also need to clarify warning states. Warnings can be part of the validation arrary or the invalidation array results.